src/platform_impl/linux/x11/util/hint.rs

```diff
@@ -93,6 +92,90 @@ impl WindowType {
     }
 }
 
+pub struct MotifHints {
+    hints: MwmHints,
+}
+
+#[repr(C)]
+struct MwmHints {
+    flags: c_ulong,
+    functions: c_ulong,
+    decorations: c_ulong,
+    input_mode: c_long,
+    status: c_ulong,
+}
+
+#[allow(dead_code)]
+mod mwm {
+    use libc::c_ulong;
+
+    pub const MWM_HINTS_FUNCTIONS: c_ulong = 1 << 0;
```

Comment:
Can you add a comment saying where you got these?

Reply:
I've added a link to the Stack Overflow answer where I found these. Is this acceptable?